src/mavedb/view_models/experiment.py

```diff
@@ -72,6 +73,12 @@ def validate_keywords(cls, v):
         keywords.validate_keywords(v)
         return v
 
+    @validator("title", "short_description", "abstract_text", "method_text")
+    def validate_field_is_non_empty(cls, v, field):
```

Comment:
```suggestion
    def validate_field_is_non_empty(cls, v):
```
Sorry, made a mistake in the suggestion earlier-- I don't think we need this third argument now since we don't use it anywhere.

Reply:
No worries. I tried `field.name` but the underscore looks weird in error messages. I agree to remove it since we don't use it anywhere.